<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Additionsprogramm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbSummand1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Erster Summand</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSummand2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zweiter Summand</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSumme">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Summe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbSummand1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbSummand2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbSumme">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btAddition">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Zusammenz√§hlen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btAddition</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addition()</slot>
 </slots>
</ui>
